### PR TITLE
A clone of the mint sign up page

### PR DESCRIPTION
**Pull request by alvinlouis29**
the goal of this project is to create a page that looks like the mint signup page at the URL below;
https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26ivid%3D6078a170-97c9-43ad-b0dd-2a1a0b833bca%26adobe_mc%3DMCMID%253D73276305084148563933419785639288950080%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1580064768%26ivid%3D6078a170-97c9-43ad-b0dd-2a1a0b833bca

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                    
+                    <h2>Create an Intuit account</h2>
+                    <div class="header-closure">
+                        <p>One account for everything Intuit, including Mint.</p>
+                        <p><a href="#">Learn more</a></p>
+                    </div>
+                    <form>
+                        <div>
+                            <p>Email address</p>
+                            <input type="text" name="email" >
```
Comment (euqueme) [inline index.html]: type="email"

File: index.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                        </div>
+
+                        <div class="space">
+                            <p>Phone (recommended)</p>    
+                            <p><input type="text" name="phone" ><br ><span class="sub-text">Standard call, messaging or data rates may apply.</span></p>
```
Comment (euqueme) [inline index.html]: type="tel"

File: index.html [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+                        </div>
+
+                        <div class="space">
+                            <p>Password</p>
+                           <input type="text" name="email" >
```
Comment (euqueme) [inline index.html]: type="password"

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Quemé P. (TSE) :woman_technologist:

#### Hello @alvinlouis29 and @adewaleK 
### Good job! everything looks nice and it feels like the original :art: also great attention to detail :microscope:  just a couple of things to work on before we can give you the approval:

- [ ] Align the **mint, quickbooks** and **turbotax** Icons vertically with the text either lower the Icons or bring up the text so it looks closer to the original

**Yours:**
image

**Original**
image

PROJECT SPECIFIC

**Implementation details**

**Middle**

- [ ] It has an **\<input\>** of type email 

- [ ] It has an **\<input\>** of type tel 

- [ ] It has an **\<input\>** of type password 

[OPTIONAL] I strongly recommend to use the **\<main\>** tag to surround the most important part of the HTML file to keep the semantic cohesive.

[OPTIONAL] Create a Live demo of your project by following these [steps](https://help.github.com/en/enterprise/2.13/user/articles/configuring-a-publishing-source-for-github-pages) and add it to your Readme file

I added some suggestions in your code to fix some of the changes you can find it in the comments below :point_down: 

### That's it. Please :pray: correct these changes and submit back for a code review when you are done :rocket:

STATUS OF THE PROJECT: REQUESTED CHANGES :vertical_traffic_light:

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status : Approved  :airplane: 

**Congratulations** Team for implementing all the changes requested by previous reviewer :+1: 
Great job :100:  
You can now merge to master and remember to submit project completion form

Happy coding :computer: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo-9738b0128/)

**Pull request merged**